solution/README.md

```diff
@@ -1,88 +1,115 @@
-# Morning challenge: implement a waterfall function
+# Solution explanation
```

Comment:
This is the only workshop I know of to have a full written explanation in the solutions folder. It's worth mentioning this in the main readme so students know it's here.

Reply:
Yeah true. I was planning on rewriting the readme at some point as well, but I'll drop a link in for now

Reply:
Done